src/cls/_ZPM/Repo/Remote/ModuleInfo.cls

```diff
@@ -1,12 +1,14 @@
-Class %ZPM.PackageManager.Core.VersionedBase Extends %ZPM.PackageManager.Core.ModuleInfo [ Abstract ]
+Class %ZPM.Repo.Remote.ModuleInfo Extends (%RegisteredObject, %ZPM.General.ModuleInfo)
 {
 
+Parameter XMLTYPE = "CoreModule";
```

Comment:
Why is a different XMLTYPE needed? Please add a comment to the parameter explaining if it is required